docs/variants/hardkernel_odroid_h4/test-matrix.md

```diff
@@ -0,0 +1,144 @@
+| 5.  | [Custom network boot entries][CNB]    | CNB           | CNB001.002                           |
+| 6.  | [UEFI compatible interface][EFI]      | EFI           | EFI001.001                           |
+| 7.  | [UEFI Shell][USH]                     | USH           | All                                  |
+| 8.  | [NVMe support][NVM]                   | NVM           | NVM001.001, NVM001.002               |
+| 9.  | [Network boot][PXE]                   | PXE           | All                                  |
+| 10. | [Display ports][DSP]                  | DSP           | DSP002.001, DSP003.001               |
+| 11. | [Audio subsystem][AUD]                | AUD           | AUD007.001, AUD008.001               |
+| 12. | [Sleep mode][SUSP]                    | SUSP          | SUSP001.001, SUSP002.001, SUSP003.001, SUSP005.001|
+| 13. | [SMBIOS verification][DMI]            | DMI           | All                                  |
+| 14. | [eMMC support][MMC]                  | MMC           | All                                  |
+| 15. | [SATA support][SATA]                  |               | All                                  |
+| 16. | [Sign of life][SOL]                   | SOL           | All                                  |
+| 17. | [Persistent Boot Splash][LCM]         | LCM           | All                                  |
+| 18. | [Debian Stable and Ubuntu LTS support][LBT] | LBT     | LBT003.001, LBT003.002, LBT004.001, LBT004.002|
+| 19. | [Power state after power fail][PSF]    | PSF           | All                                  |
+| 20. | [Dasharo Tools Suite][DTS]            | DTS           | DTS001.001, DTS002.001, DTS003.001, DTS004.001, DTS005.001, DTS006.001 |
+| 21. | [Reset to defaults][RTD]              | RTD           | RTD001.001, RTD002.001, RTD003.001, RTD004.001, RTD006.001 |
+| 22. | [Ethernet interface][NET]             | NET           | All                                  |
+| 23. | [Dasharo Configuration Utility][NET]  | DCU           | DCU001.001, DCU002.001, DCU003.001   |
+| 24. | [ESP scanning][ESP]                   | ESP           | All                                  |
+| 25. | [Network Boot Utilities][NBT]          | NBT           | All                                  |
+| 26. | [USB detection][UDT]                  | UDT           | All                                  |
+| 27. | [USB booting][UBT]                    | UBT           | All                                  |
+| 28. | [Setup Menu information][SET]         | SET           | All                                  |
+| 29. | [Ubuntu booting performance test][BUB]| BUB           | All                                  |
+| 30. | [CPU status][CPU]                     | CPU           | CPU001.001, CPU002.001, CPU003.001, CPU004.001|
+| 31. | [Auto boot time-out][BMM]             | BMM           | All                                  |
+
+[CBO]: .
```

Comment:
Wrong links, either remove and provide valid ones.

Reply:
Added links to dasharo-compatibility tests: https://github.com/Dasharo/docs/pull/903/commits/3bdfee037693ffbb579c132378a2d0cc116b033f
We are missing documentation for:
- SATA
- RTD
- SET
- BMM

Reply:
Added links to dasharo-security tests: https://github.com/Dasharo/docs/pull/903/commits/783fadbe54b91636b975dd17d16d0be95e1f6e34


- USS test id is used for both Usb stack enable/disable and Ubuntu Server Suppoert. The suites are in different tets modules, so maybe it's not a problem?

Reply:
Dasharo-stability:https://github.com/Dasharo/docs/pull/903/commits/a504e21f6112cc47d1b2cd8876e7abfb9e1a90de
- No TBD documentation in dasharo docs